yarn.lock

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
```

Comment:
This needs to be regenerated after removing the deps from the root package. Just run `yarn` in the root directory and commit the changes.